.github/workflows/check-catalyst.yaml

```diff
@@ -190,7 +180,7 @@ jobs:
       if: steps.cache-mhlo.outputs.cache-hit != 'true'
       uses: actions/cache@v3
       with:
-        path:  mlir/llvm-project
+        path: mlir/llvm-project
```

Comment:
There are some other places with two spaces in this action. Let's fix them all :sweat_smile:

Reply:
Ah shoot, I guess I still missed some ^^